apps/console/src/features/applications/models/application-inbound.ts

```diff
@@ -72,6 +80,28 @@ export interface OAuth2PKCEConfigurationInterface {
     supportPlainTransformAlgorithm?: boolean;
 }
 
+export interface ClientAuthenticationConfigurationInterface {
+    tokenEndpointAuthMethod?: string;
+    tokenEndpointAuthSigningAlg?: string;
+    tlsClientAuthSubjectDn?: string;
```

Comment:
Do we really need to make these attributes optional? Check below interfaces as well.

Reply:
Yes, all the newly added attributes are optional attributes.